graphql_server/__init__.py

```diff
@@ -230,11 +231,11 @@ def get_response(
     as a parameter.
     """
 
-    if not params.query:
-        raise HttpQueryError(400, "Must provide query string.")
-
     # noinspection PyBroadException
     try:
+        if not params.query:
+            raise HttpQueryError(400, "Must provide query string.")
```

Comment:
Why move this inside the try catch block? Won't that mean that the response will just return None?

Reply:
If you check the original code, before merging #36, the `execute_graphql_request` function was wrapped inside [execute call](https://github.com/graphql-python/graphql-server-core/pull/36/files#diff-0b0d844d3af70da5de18ae457e73d2c2L329), which also was inside the try-catch block on `get_response` function. 

The previous behaviour returned **None** if `catch_exc` was true and without putting it inside the try-catch, you will always get this error if any of the response in the list had no params, which isn't the desired behaviour.

Reply:
Oh sorry I misread the line `except catch_exc`. My bad